quickstarts/harbor/config.yml

```
@@ -0,0 +1,50 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: harbor
+
+# Displayed in the UI (required)
+title: Harbor
+
+# Long-form description of the quickstart (required)
+description: |
+  ## What is Harbor?
+
+  Harbor Registry is an enterprise-class registry server that stores and distributes container images. Harbor extends the open source Docker Distribution by adding the functionalities usually required by an enterprise, such as security, identity, and management. As an enterprise private registry, Harbor offers enhanced performance and security. 
+
+  ## Data sources for this quickstart
+
+  This quickstart was built based on `harbor` metrics sent to New Relic using Prometheus remote write.  
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  New Relic's instant observability quickstart provides alerts, multiple instance monitoring, and dashboards to detect the health and availability of servers to resolve issues before impacting end-users. By leveraging Prometheus metrics we gain insights into the behavior of your Harbor installation.
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - harbor
+  - container
+  - registry
+  - vmware
+  - prometheus
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - setup-prometheus
+
+# alerts:
+
+dashboards:
+  - harbor
+
+# Documentation references
+documentation:
+  - name: Prometheus Remote Write Configuration
+    url: https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/
+    description: Configure your Prometheus Server to remote write metrics to New Relic
+  - name: Harbor Metrics Documentation
+    url: https://github.com/goharbor/website/blob/release-2.3.0/docs/administration/metrics/_index.md
```

It might be a nicer experience to see the doc on the Harbor website:

```suggestion
    url: https://goharbor.io/docs/2.3.0/administration/metrics/
```